### ISSUE TITLE
wandb & tensorboard

### ISSUE THREAD

**Issue by ZhikangNiu** [NoFish-528/encodec-pytorch#9]
visualize the loss

**Comment by XhqGlorry11** [comment]
@ZhikangNiu Nice work! I want to reproduce your training. Can you share the training tensorboard for reference? Thx.

**Comment by ZhikangNiu** [comment]
Sorry.It has been a long time here and I couldn’t found this log. So sad, feel free to ask questions 发自我的 iPhone在 2024年10月23日，20:10，xhq11 ***@***.***> 写道：﻿
@ZhikangNiu Nice work! I want to reproduce your training. Can you share the training tensorboard for reference? Thx.

**Comment by XhqGlorry11** [comment]
@ZhikangNiu Do you still keep your original training config file and running command?

**Comment by ZhikangNiu** [comment]
actually, I couldn’t find it :(发自我的 iPhone在 2024年10月24日，16:19，xhq11 ***@***.***> 写道：﻿
@ZhikangNiu Do you still keep your original training config file and running command?
